### PR TITLE
#76 Fixed default operator not being applied in GenerateQueryVisitor

### PR DESCRIPTION
**Pull request by richardcockerill**
Fixed default operator not being applied to right hand side of group node in GenerateQueryVisitor

### CODE REVIEW

File: src/Foundatio.Parsers.LuceneQueries/Nodes/GroupNode.cs [hunk 1]
```diff
@@ -91,7 +91,7 @@ public class GroupNode : QueryNodeBase, IFieldQueryWithProximityAndBoostNode {
             builder.Append("(");
 
         if (Left != null)
-            builder.Append(Left);
+            builder.Append(Left is GroupNode ? ((GroupNode) Left).ToString(defaultOperator) : Left.ToString());
```
Comment (richardcockerill) [inline src/Foundatio.Parsers.LuceneQueries/Nodes/GroupNode.cs:94]: It's a bit ugly using a cast here and below but it's the simplest option.
An alternative is to move this entire method back into the GenerateQueryVisitor

Comment (ejsmith) [reply src/Foundatio.Parsers.LuceneQueries/Nodes/GroupNode.cs:94]: Yeah, I think it's fine.

**Review by ejsmith — APPROVED** [review]
Looks good! Thank you @richardcockerill!

**Comment by richardcockerill** [comment]
@ejsmith what's the release schedule/process to get this into a new NuGet release?

**Comment by ejsmith** [comment]
@richardcockerill just pushed a new version to nuget.

**Pull request merged**